src/coreclr/jit/lclvars.cpp

```diff
@@ -5793,8 +5822,8 @@ void Compiler::lvaFixVirtualFrameOffsets()
     assert(codeGen->isFramePointerUsed());
     if (lvaRetAddrVar != BAD_VAR_NUM)
     {
-        // For LoongArch64 and RISCV64, the RA is below the fp. see the `genPushCalleeSavedRegisters`
-        lvaTable[lvaRetAddrVar].SetStackOffset(-REGSIZE_BYTES);
+        // For LoongArch64 and RISCV64, the RA is at fp+8.
+        lvaTable[lvaRetAddrVar].SetStackOffset(REGSIZE_BYTES);
     }
 #endif // !TARGET_LOONGARCH64
```

Comment:
This can be unified with the above now it looks like.

Reply:
OK